### ISSUE TITLE
UI-TARS on Multimodal-Mind2Web

### ISSUE THREAD

**Issue by gusario** [bytedance/UI-TARS#82]
Hello! First of all thank you for your hard work! I'm really excited to test it out. However, I'm currently struggling with replication of benchmark like multimodal mind2web.
**What I've tried to do**:
1.  I've tried to use provided prompt from readme for desktop and run it on dataset examples. However, model always reply in format "Action: ..." instead of the label of element that you need to choose to perform action on
2. I've tried prompt provided in training_example.json, but I've got the same trouble as in case 1
3. I've tried using your prompt for desktop and match model answer's coordinates of action to element bboxes of element candidates on sample screenshot, but I've got awfully bad metrics with this setup. (I've read about output coordinates conversion from relative to absolute and I certainly performed this step) For example 7b-dpo model got scores like this on cross_website:
```
{
    "element_acc": 0.06769230769230769,
    "action_f1": 0.09767521367521366,
    "step_acc": 0.05846153846153846,
    "marco_element_acc": 0.06015472778625223,
    "marco_action_f1": 0.09054730691536822,
    "marco_step_acc": 0.0524261594062754,
    "error_ratio": {
        ">3": 0.8169014084507042,
        "2": 0.06338028169014084,
        "3": 0.11267605633802817,
        "1": 0.007042253521126761
    },
    "acc_per_website": {
        "tiktok.music": [
            0.11011904761904762,
            16
        ],
        "nba": [
            0.01515151515151515,
            11
        ],
        "recreation.gov": [
            0.10192307692307692,
            13
        ],
        "macys": [
            0.06013986013986015,
            11
        ],
        "tripadvisor": [
            0.010427807486631016,
            20
        ],
        "shopping.google": [
            0.0,
            15
        ],
        "bestbuy": [
            0.0734508547008547,
            16
        ],
        "cars": [
            0.0,
            13
        ],
        "stubhub": [
            0.05100529100529101,
            15
        ],
        "trip": [
            0.11504629629629631,
            12
        ]
    }
}
``` 
Which is too far from metrics that you reported. 

4. Added some few shots of examples in desktop prompt, but model still returned only Action: ...

**How can you help me?**

1. Ideally, I would ask you to provide evaluation scripts that you used to evaluate this benchmark
2. If it not possible for now, can you at least provide prompt that you used to evaluate your models on multimodal mind2web? 
3. How did you handle previous actions? 
4. Can you please provide example on some sample from this benchmark of how did your model input text looked like before tokenization?
5. Did you use this repo as foundation for your eval scripts https://github.com/OSU-NLP-Group/Mind2Web

**Comment by Lokshaw-Chau** [comment]
You need to finetune on the training set of mm mind2web first. 

However, I still cannot reproduce the results after finetuning UI-TARS-2B with the same data format in https://github.com/bytedance/UI-TARS/blob/main/training_example.json for 3epochs with learning rate 1e-4 and batch size 16. 
```
cross task
INFO:root:Operation F1: 0.6802138144704633
INFO:root:Element Acc: 0.5233285233285233
INFO:root:Step Success: 0.4862914862914863
INFO:root:Episode Success: 0.047619047619047616
INFO:root:Operation F1 cate: [0.9530251785043216, 0.46835443037974683, 0.6192618345273212]
INFO:root:Macro Ele Acc: 0.5497813714421143
INFO:root:Macro Op F1: 0.9027708394481623
INFO:root:Macro Step SR: 0.5101175308107814
```
Sincerely hope the authors can provide more details about finetuning for downstream evaluation benchmarks like number of training epochs, learning rate etc.

**Comment by Taoran-Lu** [comment]
Hi, we have recently released UI-TARS-1.5. You can try the latest version to see if the issue still exists. Besides, you can refer to the official repo of Multimodal-Mind2Web for more information.
